pymongo/logger.py

```diff
@@ -50,6 +50,7 @@ class LogMessage:
 
     def __init__(self, **kwargs: Any):
         self._kwargs = kwargs
+        # self._redact()
```

Comment:
Let's not forget to restore this

Reply:
We actually want this to be removed entirely, the redaction should only take place when the log is going to be output.